doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -619,32 +619,46 @@ file is a particular case of a module called a *library file*.
 
 .. cmd:: Declare ML Module {+ @string }
 
-   This commands dynamically loads OCaml plugins dynamically.  The string
-   argument must be a valid `findlib <http://projects.camlcity.org/projects/findlib.html>`
-   plugin name, for example ``coq-core.plugins.ltac``. The first component of
-   the plugin name is a package that has to be in scope of ``findlib``.
+   Load an OCaml "plugin" and its dependencies
+   dynamically. :n:`@string` argument must be a valid `findlib
+   <http://projects.camlcity.org/projects/findlib.html>` library name,
```

Comment:
This link is broken (it was already the case before this PR). This should fix it:
```suggestion
   <http://projects.camlcity.org/projects/findlib.html>`_ library name,
```

(assuming the link syntax works when it is split over two lines)

Reply:
Done thanks.